src/views/PuzzleCollection/PuzzleCollectionList.tsx

```diff
@@ -157,24 +157,25 @@ export function PuzzleCollectionList(): JSX.Element {
                 input: "text",
                 showCancelButton: true,
             })
-            .then((res) => {
-                const name = res.value;
-                if (!name || name.length < 5) {
-                    void alert.fire({
-                        text: _("Please provide a longer name for your new puzzle collection"),
-                    });
-                    return;
-                }
+            .then(({ value: name }) => {
+                if (name) {
```

Comment:
I think the same issue here, the empty string no longer gets the same treatment (alert: "Please provide a longer name for your new puzzle collection").  I would recommend explicitly null checking (`=== undefined`) or swal's `isConfirmed` property which I believe is more readable:

```
.then(({ value: name, isConfirmed}) => {
    if (isConfirmed) { /* do stuff */ }
});
```